src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/config/ManageRecallsApiExceptionHandler.kt

```diff
@@ -7,72 +7,57 @@ import org.springframework.http.HttpStatus.INTERNAL_SERVER_ERROR
 import org.springframework.http.HttpStatus.UNAUTHORIZED
 import org.springframework.http.ResponseEntity
 import org.springframework.security.access.AccessDeniedException
+import org.springframework.validation.FieldError
+import org.springframework.validation.ObjectError
+import org.springframework.web.bind.MethodArgumentNotValidException
 import org.springframework.web.bind.annotation.ExceptionHandler
 import org.springframework.web.bind.annotation.RestControllerAdvice
 import javax.validation.ValidationException
 
 @RestControllerAdvice
 class ManageRecallsApiExceptionHandler {
+  private val log = LoggerFactory.getLogger(this::class.java)
+
   @ExceptionHandler(ValidationException::class)
   fun handleValidationException(e: Exception): ResponseEntity<ErrorResponse> {
     log.info("Validation exception: {}", e.message)
     return ResponseEntity
       .status(BAD_REQUEST)
-      .body(
-        ErrorResponse(
-          status = BAD_REQUEST,
-          userMessage = "Validation failure: ${e.message}",
-          developerMessage = e.message
-        )
-      )
+      .body(ErrorResponse(BAD_REQUEST, "Validation failure: ${e.message}"))
```

Comment:
curious why you removed the developer message?

Reply:
This was inherited from the template.  My use of this class is the first real use that I know of, so I simplified things.  The UI, as our only client, certainly doesn't use any of these error details yet.

Reply:
ok thanks - we might find out later what's the purpose of that message...